cluster-autoscaler/core/scaledown/actuation/drain_test.go

```diff
@@ -63,24 +63,13 @@ func TestDaemonSetEvictionForEmptyNodes(t *testing.T) {
 		{
 			name:              "Successful attempt to evict DaemonSet pods",
 			dsPods:            []string{"d1", "d2"},
-			nodeInfoSuccess:   true,
```

Comment:
nit: `nodeInfoSuccess` shold also be removed from the struct above.

Reply:
Fixed.